fe/fe-core/src/main/java/com/starrocks/backup/Repository.java

```diff
@@ -214,6 +223,9 @@ public Status initRepository() {
             return new Status(ErrCode.COMMON_ERROR,
                     "Invalid repository dir. expected one repo info file. get more: " + remoteFiles);
         } else {
+            if (path == "__palo_repository_") {
+                return new Status(ErrCode.COMMON_ERROR, "Use new repository prefix");
+            }
             // repo is already exist, get repo info
```

Comment:
Can you change this comment? It seems non-existent will fall into this branch.

Reply:
have fix it